src/grt/test/congestion6.tcl

```diff
@@ -4,8 +4,8 @@ read_lef "Nangate45/Nangate45.lef"
 read_def "gcd.def"
 
 set_global_routing_layer_adjustment metal2 0.9
-set_global_routing_layer_adjustment metal3 0.9
-set_global_routing_layer_adjustment metal4-metal10 0.8
+set_global_routing_layer_adjustment metal3 0.7
+set_global_routing_layer_adjustment metal4-metal10 1.0
```

Comment:
Why change these?

Reply:
To keep the same behavior from before. I don't want resources in the layers above metal3 to create a high congestion scenario. With the previous setting, the 1-track minimum created enough resources to avoid congestion.